scripts/spcgeonode/django/Dockerfile-dev

```diff
@@ -0,0 +1,37 @@
+#--------- Generic stuff all our Dockerfiles should start with so we get caching ------------
+# Note this base image is based on debian
+
+FROM geonode/spcgeonode:django-3.0
+MAINTAINER Irwan Fathurrahman<meomancer@gmail.com>
+
+# This section taken on 2 July 2015 from
+# https://docs.docker.com/examples/running_ssh_service/
+# Sudo is needed by pycharm when it tries to pip install packages
+RUN apt-get update && apt-get install -y openssh-server sudo
+RUN rm -r /var/run/sshd
```

Comment:
this should be removed because it will create error:
```
Step 4/16 : RUN rm -r /var/run/sshd
 ---> Running in 748f1ff71976
rm: cannot remove '/var/run/sshd': No such file or directory
ERROR: Service 'dev' failed to build: The command '/bin/sh -c rm -r /var/run/sshd' returned a non-zero code: 1
```